src/Storages/MergeTree/KeyCondition.h

```diff
@@ -414,6 +415,8 @@ class KeyCondition
         RPNElement & out,
         size_t & out_key_column_num);
 
+    bool unknownOrAlwaysTrue(bool unknown_any) const;
```

Comment:
Cannot understand what does it mean - need comments.

Reply:
Need to add explanation "why" with examples, around fields `strict` and `single_point`.